npm-packages/docs/docs/quickstart/nextjs.mdx

```diff
@@ -66,8 +66,8 @@ version of this quickstart.
   </Step>
   <Step title="Install the Convex client and server library">
     To get started, install the `convex`
-    package which provides a convenient interface for working
-    with Convex from a React app.
+    package, which provides a convenient interface for working
+    with Convex.
```

Comment:
```suggestion
    package.
```
agree "from a React app" wasn't very useful, and now the rest doesn't seem useful either IMO